README.md

```diff
@@ -13,7 +13,7 @@ Juju-verify requires Juju 2.8.10 or higher.
 * nova-compute (Usable with the next stable release of the charm. Currently available as a [nova-compute-rc])
 * ceph-osd (Usable with the custom release of the charm in rgildein namespace. [cs:~/rgildein/ceph-osd-0] and [cs:~/rgildein/ceph-mon-3])
 * ceph-mon (Usable with the custom release of the charm in rgildein namespace. [cs:~/rgildein/ceph-mon-3])
-* neutron-gateway (Usable with the custom release of the charm in rgildein namespace. [cs:~/martin-kalcok/neutron-gateway-0])
+* neutron-gateway (Usable with the custom release of the charm in rgildein namespace. [cs:~/martin-kalcok/neutron-gateway-1])
```

Comment:
After looking at the conflicts I will have with this PR, I found out:
 - You need change this also at line 60
 - You need to change this in [changelog](https://github.com/canonical/juju-verify/blob/master/docs/changelog.rst)
 - You need to change this in [getting-started](https://github.com/canonical/juju-verify/blob/master/docs/getting-started.rst)
All these changes are necessary due to the change in the minimum revision.